server/src/test/java/org/elasticsearch/rest/action/cat/RestIndicesActionTests.java

```diff
@@ -170,4 +188,19 @@ private IndicesStatsResponse randomIndicesStatsResponse(final Index[] indices) {
             shardStats.toArray(new ShardStats[shardStats.size()]), shardStats.size(), shardStats.size(), 0, emptyList()
         );
     }
+
+    private <T> T[] removeRandomElement(T[] array) {
```

Comment:
maybe you can use a List and avoid all the ceremony need to remove an element from an array?